hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -991,8 +991,8 @@ void checkOnlineRegionsReportForMeta(final ServerName serverName, final Set<byte
         final RegionInfo hri = getMetaRegionFromName(regionName);
         if (hri == null) {
           if (LOG.isTraceEnabled()) {
-            LOG.trace("Skip online report for region=" + Bytes.toStringBinary(regionName) +
-              " while meta is loading");
+            LOG.trace("Skip online report for region={} while meta is loading from server={}",
```

Comment:
region= is redundant.

Reply:
So is server= ... smile. Who else would be making a report here.

Reply:
Why would the server making the report be redundant?

Reply:
If regionName is redundant then why does `getMetaRegionFromName`  take it as an argument?